utils/misc.js

```diff
@@ -39,6 +39,22 @@ export default class Miscellaneous {
 
         // the site might be behind a proxy.
         expressApp.enable('trust proxy');
+
+        // disable bot crawling, without file.
+        expressApp.get('/robots.txt', (req, res) => {
+            res.type('text/plain');
+            res.send("User-agent: *\nDisallow: /");
+        });
```

Comment:
Lets remove this & instead add a `robots.txt` file in the `public` directory. This gives more control to the developer.

Reply:
Done!